sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -291,8 +292,17 @@ def GetHeaders(  # pylint: disable=too-many-statements,too-many-branches
             if_none_match_value = options["continuation"]
         elif options.get("isStartFromBeginning") and not options["isStartFromBeginning"]:
             if_none_match_value = "*"
+        elif options.get("startTime"):
+            dt = options.get("startTime")
+            # If datetime has specified timezone, and it's not utc, convert to utc.
+            # Otherwise, assume it is in UTC.
+            if isinstance(dt, datetime):
+                if dt.tzinfo is not None and dt.tzinfo.utcoffset(dt) is not None:
+                    dt = dt.astimezone(timezone.utc)
```

Comment:
It may be worthwhile expanding on this comment here, typically naiive datetimes (lacking tzinfo) are presumed to be in the system timezone not UTC - this is why 'utcnow' is deprecated in the Python SDK as it can lead to confusion.

If so, it may be worthwhile calling astimezone unconditionally- as then it would correctly adjust naiive timezones.

Reply:
I agree, I have made the recommended changes.